src/models/BookAuthor.js

```diff
@@ -0,0 +1,5 @@
+import { sequelize } from "../utils/database.js";
+
+const BookAuthor = sequelize.define('BookAuthor', {});
```

Comment:
is something missing here? should this model have fields?

Reply:
no this model is kept just for relation purpose as books and authors are many to many relation so they are joined by BookAuthor named table